Cargo.toml

```diff
@@ -986,6 +986,14 @@ name = "Meshlet"
 description = "Meshlet rendering for dense high-poly scenes (experimental)"
 category = "3D Rendering"
 wasm = false
+setup = [
```

Comment:
Is there a place we can document this new options? Perhaps the `docs` folder?

Reply:
I think this probably belongs in the example-showcase somewhere? I'm not sure, but I would also like better docs.

Reply:
documented on the struct in the example showcase script for now